scripts/assets.gd

```diff
@@ -117,3 +117,6 @@ func getsexfeatures(person):
 	###---End Expansion---###
 ###---End Expansion---###
 
+func getrandomeyecolor():
```

Comment:
This function is a regression, eye colors are now stored in globals.gd. Changing the eye colors in quests from 'blond' to 'amber' should happen there.

Reply:
Fixed I think. I tied the text array of eye colors to the one in globals.

Reply:
Then the function should simply be removed, it already exists in vanilla

Reply:
Yeah I did, its removed in next update.